src/goto-analyzer/static_verifier.h

```diff
@@ -9,6 +9,8 @@ Author: Martin Brain, martin.brain@cs.ox.ac.uk
 #ifndef CPROVER_GOTO_ANALYZER_STATIC_VERIFIER_H
 #define CPROVER_GOTO_ANALYZER_STATIC_VERIFIER_H
 
+#include <goto-checker/properties.h>
+#include <goto-programs/abstract_goto_model.h>
```

Comment:
Wouldn't a forward declaration be sufficient?

Reply:
Do you mean just for `abstract_goto_modelt`?

Reply:
Yes